### PR TITLE
Disable docstring parsing for external tests

### PR DESCRIPTION
**Pull request by gpetiot**
and fix one of the tests from the `ocaml` repo testsuite

### CODE REVIEW

**Comment by jberdine** [comment]
My impression was that the detection of project root had some problem. What do you think?

**Comment by gpetiot** [comment]
I checked that the detection of the project root is fine when the option `disable-outside-detected-project` is set, but it disables the formatting if there is no `.ocamlformat` file at the root (and this is the case for `infer`) so I decided to override the options with configuration files instead.

File: test/.ocamlformat [hunk 1]
```diff
@@ -0,0 +1 @@
+parse-docstrings = false
```
Comment (jberdine) [inline test/.ocamlformat]: This seems too specific, the same problem will happen for any other option. Perhaps set profile = default instead?

**Comment by gpetiot** [comment]
It should be good to go now

**Pull request merged**